src/funnel.hpp

```diff
@@ -9,6 +9,9 @@
 #include <functional>
 #include <iostream>
 #include <limits>
+#include "aligner.hpp"
+#include "mapper.hpp"
```

Comment:
These headers can't both be newly necessary here. I think we just need the vg Protobuf header, for `Alignment`.